### PR TITLE
Kettanaito update msw comparison

### PR DESCRIPTION
**Pull request by samselikoff**

### CODE REVIEW

**Comment by samselikoff** [comment]
I meant to push these to your PR but clearly I don't know how to use GitHub 🙃

@kettanaito let me know what you think! Sorry again for the delay on this one.

File: src/routes/docs/comparison-with-other-tools.mdx [hunk 1]
```diff
@@ -55,25 +57,7 @@ Some other advantages Mirage has:
...
-    ReactDOM.render(<App />, document.getElementById("root"))
-  }
-
-  bootstrap()
-  ```
-
-  The service worker, while enabling usage of devtools, does add a bit more complexity to the overall integration. For example you also need to ensure your service worker [updates on source code reloads](https://redd.gitbook.io/msw/getting-started#chrome).
-
-- Because there is no service worker barrier, it's trivial to interact with Mirage's server and DB during development. People have even made dev-time tools to tweak their server at runtime – for example, a floating React component with a slider bound to Mirage's timing delays.
+  MSW requires one additional step - you need to generate a file that in the public directory of your project, which you might also want to strip from production builds.
```
Comment (kettanaito) [inline src/routes/docs/comparison-with-other-tools.mdx:60]: Could we mention that it's a `Service Worker file`? Just to be clear what the sentence is about.

**Comment by kettanaito** [comment]
Love the changes! Got a little suggestion.

I'm usually adding a person's remote into my own, and push to their feature branch:

```bash
$ git remote add sam https://github.com/miragejs/site.git
$ git fetch sam
$ git checout sam/<BRANCH_NAME>
$ git add . && git commit -m 'Some adjustments'
$ git push sam HEAD:<BRANCH_NAME>
```

**Comment by samselikoff** [comment]
Gotcha, now to update your PR I would have needed to push to `update-msw-comparison` on your fork right? Could I do that directly or would I need to do it via another PR to your fork?

(Also I added the wording tweak, if you think it looks good we can merge this!)

**Comment by kettanaito** [comment]
Thanks you! Yes, the change looks amazing. 

No need to create a separate pull request, you should be able to push directly to the feature branch on my remote (`update-msw-comparison`), using the code I've mentioned above. Feel free to merge this, or push to another PR and merge that one 🙏 Thank you for an operative response.

Also, please let me know if the comparison with MirageJS on the MSW's side makes sense to you. I'd be glad to change it so it reflects the difference better/

**Comment by samselikoff** [comment]
Went ahead and merged because GH automatically added the Co-Authored By so you get credit too :)

Thanks again!

I like the comparison on MSW's site, the only thing is there are teams using Mirage with GraphQL (Heroku has been for years) so it's not quite right to say it doesn't work, just that there's no additional API helpers the way there are for Rest. Though I think it might not look that different from MSW once you set it up with the `graphql` library.

There's also [this](https://github.com/kloeckner-i/ember-cli-mirage-graphql) that's used a bit in the Ember community.

All that being said it doesn't bother me that you have it, we should really have better graphql support and in fact some folks are working on it! So maybe once we have something in the guides/tutorial about GraphQL, I will be able to ping you about changing that part of the table :)

Thanks again and keep up the great work on msw!

**Comment by kettanaito** [comment]
Thanks once more on updating this! I'd be glad to update that comparison, just let me know once the updates are out :) Mirage is awesome, and I'm looking forward to new features in both libraries!

**Pull request merged**